html/janus.js

```diff
@@ -100,6 +199,13 @@ Janus.init = function(options) {
 			}
 		}
 		Janus.log("Initializing library");
+
+		var usedDependencies = deps || options.dependencies || Janus.useDefaultDependencies();
+		Janus.isArray = usedDependencies.isArray;
+		Janus.webRTCAdapter = usedDependencies.webRTCAdapter;
+		Janus.httpAPICall = usedDependencies.httpAPICall;
+		Janus.checkJanusExtendsion = usedDependencies.checkJanusExtendsion;
```

Comment:
It seems a bit weird to hang these off the Janus object. I know the ```dependencies``` key was removed by request, just wondering if there's an even cleaner way to do this?

Reply:
Well, you could have people pass the deps to both `Janus.isExtensionEnabled` and `new Janus()`. In that case the function call argument (dependencies) could simply be captured & used, but it would break backwards compatibility of `Janus.isExtensionEnabled()`. So I don't think that's such a good idea.